content/en/blog/_posts/2023-10-10-cri-o-community-package-infrastructure.md

```diff
@@ -174,7 +174,7 @@ CRI-O. A [test pipeline](https://github.com/cri-o/packaging/actions/workflows/ob
 in the OBS workflow ensures that the packages can be correctly installed and
```

Comment:
I think the 'under the hood' phrase could be changed.

ref:https://kubernetes.io/docs/contribute/style/style-guide/#avoid-jargon-and-idioms

Reply:
Take care using our docs style guide to review blogs. For a blog, which has an identified author, it's OK for the author to write as themselves.

Reply:
>Take care using our docs style guide to review blogs. For a blog, which has an identified author, it's OK for the author to write as themselves.

Oops, I didn't know that.